### ISSUE TITLE
Add metric to prevent overfitting

### ISSUE THREAD

**Issue by kxl4126** [kxl4126/polynomial-regression#1]
Adding degrees to the polynomial could lead to overfitting. I need to add a metric to track where the decrease in RMSE is not worth the increase in degree of the polynomial and risk of overfitting. This can be done by graphing the RMSE and using the informal "elbow method" to find the optimal hyperparameter.
